src/OrganisationUnitTree/useChildren.js

```diff
@@ -0,0 +1,14 @@
+import { useMemo } from 'react'
+
+export const useChildren = ({ children }, path, orgUnitsPathsToInclude) =>
```

Comment:
You are not using `path` so probably shouldn't pass it to this function in the first place.